### PR TITLE
Fixed #27346 -- Removed the Content-Length response header in ConditionalGetMiddleware

### PR DESCRIPTION
**Pull request by adammalinowski**

### CODE REVIEW

File: docs/ref/middleware.txt [hunk 1]
```diff
@@ -181,8 +181,6 @@ header, the middleware adds one if needed. If the response has a ``ETag`` or
 ``If-Modified-Since``, the response is replaced by an
 :class:`~django.http.HttpResponseNotModified`.
 
-Also sets ``Content-Length`` response-header.
-
```
Comment (charettes) [inline docs/ref/middleware.txt]: It might be worth changing this to a `versionchanged:: 1.11` by mentioning it was removed.

**Comment by charettes** [comment]
Thanks and please uncheck _Patch needs improvement_ on the ticket when you are done.

File: docs/ref/middleware.txt [hunk 2]
```diff
@@ -181,13 +181,14 @@ header, the middleware adds one if needed. If the response has a ``ETag`` or
...
 .. versionchanged:: 1.11
 
     In older versions, the middleware set the ``Date`` header and didn't set
```
Comment (timgraham) [inline docs/ref/middleware.txt]: "set the Content-Length and Date headers"

File: docs/releases/1.11.txt [hunk 3]
```diff
@@ -673,3 +673,7 @@ Miscellaneous
   ``ETag`` header to responses regardless of the setting. ``CommonMiddleware``
   and ``django.utils.cache.patch_response_headers()`` will no longer set ETags
   when the deprecation ends.
+
+* :class:`~django.middleware.http.ConditionalGetMiddleware` no longer sets the
```
Comment (timgraham) [inline docs/releases/1.11.txt]: Modify the existing note: "* ``ConditionalGetMiddleware`` no longer sets the ``Date`` header as Web
  servers set that header." This section is actually "deprecated features".

**Comment by timgraham** [comment]
buildbot, add to whitelist.

**Comment by timgraham** [comment]
merged in 37809b891eae2e73c7e80fbf5d8b9a7e86e82b54, thanks!

**Pull request closed without merging**